lighthouse-core/test/gather/gather-runner-test.js

```diff
@@ -225,6 +225,60 @@ describe('GatherRunner', function() {
     });
   });
 
+  it('clears the network cache and origin storage', () => {
+    const asyncFunc = () => Promise.resolve();
+    const tests = {
+      calledDisableNetworkCache: false,
+      calledClearStorage: false,
+    };
+    const createCheck = variable => () => {
```

Comment:
yeah, I'd be fine with that too.

Honestly I hate these tests involving the driver. Rely too much on knowledge of internal implementation instead of externally observable behavior. If we're going to assume the driver calls made and the order they're made in, I'd rather we just setup a debugger protocol record/replay system :)